Main.java

```diff
@@ -0,0 +1,16 @@
+public class Main
+ {
+    public static void main(String[] args)
+      {
+        if (args.length > 0) 
+           {
+             System.out.println(args[0]);
+           }
+        else
+             System.out.println("No arguments given");
+      }
+  }       
+	
+
+
+       
```

Comment:
Remove the unnecessary empty new lines (except of 1 new line) at the end of the file.